mandrill-api.gemspec

```diff
@@ -1,12 +1,12 @@
 Gem::Specification.new do |s|
     s.name = 'mandrill-api'
-    s.version = '1.0.54'
+    s.version = '1.0.55'
```

Comment:
Why the version change?

Reply:
2 reasons for that:

- it helps updating internals with the Gemfile
- Apache license 2.0 wants you to indicate all changes made, so updating the version is a nice way to write changelog in the new CHANGELOG.md file

Reply:
Ok.